runtime/src/lib.rs

```diff
@@ -810,6 +811,54 @@ impl lockdrop::Config for Runtime {
 	type WeightInfo = crate::weights::lockdrop::WeightInfo<Self>;
 }
 
+parameter_types! {
+	pub const TombstoneDeposit: Balance = deposit(
+		1,
+		sp_std::mem::size_of::<contracts::ContractInfo<Runtime>>() as u32
+	);
+	pub const DepositPerContract: Balance = TombstoneDeposit::get();
+	pub const DepositPerStorageByte: Balance = deposit(0, 1);
+	pub const DepositPerStorageItem: Balance = deposit(1, 0);
+	pub RentFraction: Perbill = Perbill::from_rational_approximation(1u32, 30 * DAYS);
```

Comment:
This number is known to be "random" i.e. no specific reason it is set this way.

@athei any new thoughts on what this value should be?